model/src/main/resources/reference.conf

```diff
@@ -16,6 +16,13 @@ ort {
   analyzer {
     allowDynamicVersions = true
 
+    # A list of disabled package managers. Disabling a package manager here overrides enabling it in
+    # `enabledPackageManagers`.
+    disabledPackageManagers = [Maven, NPM]
```

Comment:
Nit: Due to precedence explained in the comment, should we maybe make `disabledPackageManagers` go after `enabledPackageManagers` to emphasize that "the last word counts"?